### PR TITLE
feat: enable lightmode on multiple form pages

### PR DESCRIPTION
**Pull request by lstocchi**
### What does this PR do?

This PR adds light mode to almost all form pages. There are pages that are not here bc they are a bit more complex and it would be easier to review them separately. All you find here required basic changes and most of them have similar design.

### Screenshot / video of UI

i won't include any screenshot as they are a lot, it's more convenient you check them manually by running desktop.

### What issues does this PR fix or reference?

it is part of #7214 

### How to test this PR?

The pages to verify:

1. `ExportContainer`: go to containers page -> click on the "three vertical dots" icon of a container -> export container
2. `ImportImage`: go to the images page -> click on import (top right)
3. `LoadImages`: go to the images page -> click on load(top right)
4. `PullImage`: go to the images page -> click on pull(top right)
5. `RunImage`:  go to the images page -> click on the start action of an image
6. `SaveImages`: go to images page -> click on the "three vertical dots" icon of an image-> save image
7. `DeployToKube`: go to the pods page -> click on the "three vertical dots" icon of an image-> deploy to kube
8. `PodCreateFromContainers`: go to containers page -> select 2 or more containers -> click on `create pod` on the top
9. `CreateVolume`: go to volumes page -> click on create (top right)

- [x] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: packages/main/src/plugin/color-registry.ts [hunk 1]
```diff
@@ -1172,5 +1178,28 @@ export class ColorRegistry {
...
+
+  protected initFormPage(): void {
+    const formPage = 'formpage-';
+
+    this.registerColor(`${formPage}bg`, {
+      dark: colorPalette.charcoal[900],
+      light: colorPalette.gray[50],
+    });
+
+    this.registerColor(`${formPage}card-bg-hover`, {
```
Comment (lstocchi) [inline packages/main/src/plugin/color-registry.ts]: Noticed i removed the wrong class when creating the PR, updating

Comment (lstocchi) [reply packages/main/src/plugin/color-registry.ts]: done

**Review by benoitf — APPROVED** [review]
do you think it needs to be backported ?

**Comment by lstocchi** [comment]
@benoitf idk, up to you and @deboer-tim . If we want to backport this we also need https://github.com/containers/podman-desktop/pull/7538

**Comment by benoitf** [comment]
on my side I would say that as it was an experimental feature, no backport

**Review by deboer-tim — APPROVED** [review]
Nice work. Thanks for cleaning up a lot of hard-coded values.

**Pull request merged**